examples/studio/chat.py

```diff
@@ -8,11 +9,12 @@
     ChatMessage(text="I am having trouble signing up for your product with my Google account.", role=RoleType.USER),
 ]
 
+
 client = AI21Client()
```

Comment:
Might be worth adding a comment here saying that this is deprecated and reference to the new method